chiboycalix/time-series-chart
# Complete project implementation

#### Description of Task to be completed?
> - Write a function that generates `50` random positive integers between 1 and 15 (or between a range of your choice)
> - Use the generated data to create a chart
> - Also, generate signal data based on the generated random data above
> - Create another chart(Signal Chart) using the newly generated signal data
> - Set a default Threshold of 10 or any number you want
> - Be able to change the Threshold value and see how it affects your signal chart in real time
#### How should this be manually tested?
> - clone the repository using the command `git clone [repo link]`
> - run `git checkout feature-full-project-implementation` to checkout to this branch which has the full project implementation
> - run `npm install to install all project dependencies`
> - run `npm start` to run the app locally
> - App should automatically open in your browser
#### Screenshots (if appropriate)
<img width="1680" alt="Screen Shot 2022-03-28 at 3 34 56 PM" src="https://user-images.githubusercontent.com/35861980/160421974-b427c2ab-4732-4a6a-bc20-d28b8b5851b0.png">
